internal/cluster/cluster_service.go

```diff
@@ -95,12 +99,15 @@ func (s *Service) MemberAddrs() []pubcluster.Address {
 	return s.membersMap.MemberAddrs()
 }
 
-func (s *Service) RandomDataMember() *pubcluster.MemberInfo {
-	return s.membersMap.RandomDataMember()
+func (s *Service) RandomReplica(n int) (pubcluster.MemberInfo, bool) {
```

Comment:
Replica is related to partition table normally except CRDT Pn Counter. 
Like a partition has 7 replicas replica 0 is the owner and the others are backups. And this is related to all partitioned Data Structures. 
`Replica` in the PnCounter seems to have a different logic which has no relation to the partition table. It works over member-list only. 
When we put `replica` name in the cluster service it will be confusing for readers of the code. 
Can we somehow keep this logic in PNCounterProxy so that it will not be confusing.

Reply:
I see. I've put those methods in the cluster service in order not to copy all members. Would it be OK to add `PNCounter` somewhere in the method names? Such as: `RandomPNCounterDataMember` ?

Reply:
Moved them to PNCounter at: https://github.com/hazelcast/hazelcast-go-client/pull/648/commits/cfbd7a892b479e95c017e1cdae733ee5a09edcfc